unused_deps/unused_deps.go

```diff
@@ -221,11 +221,11 @@ func printCommands(label string, deps map[string]bool) (anyCommandPrinted bool)
 		for _, elem := range li.List {
 			for dep := range deps {
 				str, ok := elem.(*build.StringExpr)
-				if ok && edit.ShortenLabel(str.Value, pkg) == edit.ShortenLabel(dep, pkg) {
+				if ok && edit.LabelsEqual(str.Value, dep, pkg) {
```

Comment:
Now it's more about cosmetics and using in buildozer exact label, as specified in BUILD file.